source/extensions/filters/network/echo/BUILD

```diff
@@ -24,9 +25,10 @@ envoy_cc_library(
     ],
 )
 
-envoy_cc_library(
+envoy_cc_extension(
     name = "config",
     srcs = ["config.cc"],
+    security_posture = "unknown",
```

Comment:
The echo filter was the first filter I wrote in 2015. It's basically an example. Should we just nuke it from the repo? I don't think it provides any value anymore.

Reply:
Sure, SGTM, less code is always better than more. What should the new "this is a dead simple filter to get you started" example be? Or do we use envoy-filter-example repo?